libs/partners/upstage/langchain_upstage/parsers.py

```diff
@@ -0,0 +1,342 @@
+import io
+import json
+import os
+from typing import Dict, Iterator, List, Literal, Optional
+
+import fitz  # type: ignore
+import requests
+from fitz import Document as fitzDocument
+from langchain_community.document_loaders.base import BaseBlobParser
+from langchain_community.document_loaders.blob_loaders import Blob
+from langchain_core.documents import Document
+
+LAYOUT_ANALYZER_URL = "https://api.upstage.ai/v1/document-ai/layout-analyzer"
+
+LIMIT_OF_PAGE_REQUEST = 10
```

Comment:
```suggestion
DEFAULT_PAGE_BATCH_SIZE = 10
```

사용되는 코드를 보니 이 이름이 더 적합할것 같은데 어떤가요?